tcmur_cmd_handler.c

```diff
@@ -745,6 +745,18 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 
 	track_aio_request_start(rdev);
 
+	/*
+	 * Try to passthrough the cmds to the device.
+	 */
+	if (rhandler->handle_cmd) {
+		ret = handle_passthrough(dev, cmd);
+		if (ret != TCMU_NOT_HANDLED)
```

Comment:
Just one confusion about the nr_threads > 1, does that mean a handler couldn't do its own aio ?